### DISCUSSION TITLE
Rancher Desktop accessible from public URL

### DISCUSSION THREAD

**Discussion by flexguse** [rancher-sandbox/rancher-desktop#2228] [Q&A] [answered]
Windows Version: 10
Rancher Desktop Version: 1.3.0
using Kubernetes, Traefik and Ingress

Hi everybody,

in my development setup I need to have my application accessible on a public URL to test interoperability with an external service.

Let's say the URL is https://mysubdomain.mydomain.de. Traffic is routed to my local Windows dev device. If I start an application locally from my IDE (Rancher Desktop is not running) it is accessible by this domain.

If I put the application in Rancher Desktop and configure an Ingress for mysubdomain.mydomain.de I get a Gateway Timeout.

If I put mysubdomain.mydomain.de in the Window's hosts file, my application is available.

Are there any Firewall rules or network settings in Rancher Desktop which do not allow requests from any other source than localhost/127.0.0.1?

Any hint is appreciated,
Christoph

**Comment by adamkpickering** [comment, marked as answer]
Unfortunately Rancher Desktop does not currently support this use case. The problem seems to be that WSL won't listen on 0.0.0.0.

For more info please see https://github.com/rancher-sandbox/rancher-desktop/issues/1180. PRs are always welcome :)

**Reply by flexguse** [reply]
Hi Adam,

thanks a lot for your help! The WSLHostPatcher https://github.com/CzBiX/WSLHostPatcher mentioned in your linked discussion works for me, my development can proceed!

Cheers,
Christoph
